code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -403,7 +403,7 @@
 /obj/effect/golemrune/attack_hand(mob/living/user)
 	var/mob/dead/observer/ghost
 	for(var/mob/dead/observer/O in src.loc)
-		if(!O.client)	continue
+		if(!O.client && !jobban_isbanned(O, "catban"))	continue
```

Comment:
Two things:

1. This means that if the observer isn't a client AND they are catbanned then it won't let them play as a golem. I take it this should be "OR they're catbanned"

2. Should we also exclude cluwnebanned people too?